Makefile

```diff
@@ -44,6 +44,12 @@ serve: DSN_HOST ?= "127.0.0.1"
 serve: ## Sets up the server locally with default options
 	go run -tags=prod main.go --dsn_user="$(DSN_USER)" --dsn_password="$(DSN_PW)" --dsn_host="$(DSN_HOST)"
 
+.PHONY: serverless
+serverless:
+	GOOS=linux CGO_ENABLED=0 go build -o cribbage-lambda -tags=prod,lambda main.go
```

Comment:
:cactus: :book: we should either call it `lambda` or `serverless`. I'm in favor of calling it `lambda`, since we're targeting that tech